tests/type_inference/test_literals.py

```diff
@@ -78,6 +84,58 @@ def test_heterogeneous_dict(dictionary):
     assert [Dict[Any, Any]] == result
 
 
+@hs.composite
+def string_and_index(draw):
+    xs = draw(INDEX_TYPES)
+    i = draw(hs.integers(min_value=0, max_value=len(str(xs)) - 1))
+    return ["\"" +str(xs) + "\"" + "[" + str(i) + "]", i]
```

Comment:
Also you can return a tuple, not a list. The subscripting below should still work.